src/resources/swagger.yaml

```diff
@@ -19,6 +19,48 @@ paths:
               schema:
                 type: object
 
+  /submission/persistedConfig/submissionDisabled:
+    get:
+      tags:
+        - Persisted Configurations
+      summary: Get sample registration and clinical submission state
+      responses:
+        '200':
+          description: 'Sample registration and Clinical Submission system disabled state'
+    post:
+      tags:
+        - Persisted Configurations
+      summary: Disable sample registration and clinical submission
+      parameters:
+        - name: setDisabled
+          description: the boolean state to set for submissionDisabled
+          in: query
```

Comment:
usually post is to create new stuff, if you want to update it should be put / patch and the data should be in the body